heartbeat/ipsec

```diff
@@ -116,6 +123,16 @@ ipsec_stop() {
 	ipsec auto --down "${OCF_RESKEY_tunnel}"
 	local return_code=$?
 	ocf_log info "${OCF_RESOURCE_INSTANCE} : Put down tunnel ${OCF_RESKEY_tunnel} with return code ${return_code}"
+	ipsec whack --listen &>> /tmp/ipsec-agent.log
```

Comment:
Writing to a random file in `/tmp` doesn't seem very clean to me... I'm not sure if it ought to be a file in `$HA_RSCTMP` or what, maybe it ought to be a configurable parameter?

Reply:
Seems like it's the same in the start-action, so I guess we should improve it there as well.

Reply:
I was using that for debugging. Let me remove that.